README.md

```diff
@@ -2,34 +2,58 @@
 
 [![CircleCI](https://circleci.com/gh/commercetools/ctp-smoke-tests.svg?style=svg)](https://circleci.com/gh/commercetools/ctp-smoke-tests)
 
-## Prerequisites
+## How to run the smoke tests
+
 The following environment variables have to be defined to run the tests:
 
 ```
-auth-url: ${?AUTH_URL}
-client-id: ${CLIENT_ID}
-client-secret: ${CLIENT_SECRET}
-project-key: ${PROJECT_KEY}
+$ AUTH_URL=<auth url>/oauth/token
```

Comment:
@akovv It'd be good not have to give `/oauth/token`.
It'd be easier for the user, and more flexible for us.
We can use other endpoints as well in the future.

Reply:
Do you mean not specify env. vars explicitly?

Reply:
No I'd prefer to have to setup the variable like this:
```
$ AUTH_URL=<auth url>
```
(without the `/oauth/token`)

Reply:
Token is not mentioned in README on master, but yeah I agree.